searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -174,6 +180,12 @@ searxng.ready(function () {
           des: 'select next search result',
           cat: 'Results'
         },
+        'y': {
+          key: 'y',
+          fun: copyURLToClipboard,
+          des: 'copy URL of the selected result to the clipboard',
```

Comment:
```suggestion
          des: 'copy url of the selected result to the clipboard',
```
for consistency, lowercase is probably the better choice here too.